README.md

```diff
@@ -57,7 +59,7 @@ resources:
   source:
     uri: https://github.com/cloudfoundry-community/simple-go-web-app.git
 
-- name: resource-deploy-web-app
```

Comment:
It's more of a mental model kind of thing than anything else. Resources aren't meant to just config; they represent objects in your pipeline, and this resource is meant to represent a single pushed app (it should really be called `cf-app-resource`).

For example, if you wanted a downstream job to run integration tests against an app you pushed to AWS, we'd just have to implement `/in` and have it give you the URL, and you'd have a job that is triggered by that individual app changing. This workflow would be impossible with one global `pws` resource as you'd be running the wrong integration suite depending on which app was actually pushed.